src/neuroconv/tools/nwb_helpers/_configuration_models/_zarr_dataset_io.py

```diff
@@ -130,3 +134,36 @@ def get_data_io_kwargs(self) -> dict[str, Any]:
             compressor = False
 
         return dict(chunks=self.chunk_shape, filters=filters, compressor=compressor)
+
+    @classmethod
+    def from_neurodata_object(
```

Comment:
there's a lot of repeated code here with the hdf5 version of this function. Would there be any way to reduce the repetition?

Reply:
Yep, added a staticmethod to the base class that should help de-duplicate.